### ISSUE TITLE
Drone Block placement appears to be skewed along the north/south axis

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#609]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.16.1
### Forge Version
32.0.108
### Mod Version
1.16.1-2.2.0-29
### Describe your problem, including steps to reproduce it

When building spheres and circles with drones, I've noticed that they appear to stretch the shape in the north/south axis, resulting in placements such as these:

GPS preview
![2020-09-09_21 30 06](https://user-images.githubusercontent.com/9543430/92671380-18c12d80-f2e4-11ea-8459-cf990769362e.png)

Actual Placement by Drone:
![2020-09-09_21 31 38](https://user-images.githubusercontent.com/9543430/92671402-2971a380-f2e4-11ea-8e38-81c410f9c39f.png)

Seems to do it with all shapes, here's a box with the real corners highlighted:
![2020-09-09_21 36 18](https://user-images.githubusercontent.com/9543430/92671554-95540c00-f2e4-11ea-99c9-9d63783e7389.png)

### Any other comments?

https://gist.github.com/MuteTiefling/facc4f0a0efabbd3bc4ed16c92dc1482

**Comment by desht** [comment]
Hmm, has the feel of an arithmetic rounding error.  Will investigate (might take a few days due to RL).

**Comment by desht** [comment]
Had a chance to test, but so far I haven't been able to reproduce this problem.  (Just tested with a Y-axis cylinder of one size so far, so it might be size related... anyway I'll try a few other combinations when I get a chance).

**Comment by MuteTiefling** [comment]
No rush. I'll go back and test again with a lighter pack. I just remembered my test pack had KubeJS in it and it was doing some funky things to other mods. So I'll see if I can reproduce without it. Otherwise the only other mods in this pack are Botania and Mekanism.

**Comment by MuteTiefling** [comment]
Well, unfortunately KubeJS wasn't interfering. It doesn't seem 100% consistent either, as the first test I did actually came out fine. It didn't break until I went back and did a cylinder or sphere. 

Also, thinking it might be something to do with coordinates, I went to 0,0 and then did in a test in each quadrant. Unfortunately, this didn't result in any difference. All four of my rings turned out stretched in the same way. 

I did build a new program to make testing it a bit easier. Just needs one coordinate now, the chest with the input materials:

https://gist.github.com/MuteTiefling/c4c19193141522f38acdcf08b5fa19b0

This is using a radius of 10. Also tried a radius of 50 and got a similar result. 

In any case, no need to rush at all. It's an odd one, but probably lower priority. If you want to share what you've got in your test environment, though, I could run more tests with that to see if adding something else is actually fixing it.
